### PR TITLE
Added handler for "Never Stop" comment

### PR DESCRIPTION
**Pull request by MortenTobiasNielsen**
I need this for easier TAS development. It should be changed to be a modifier when we have that functionality.

### CODE REVIEW

**Review by theis999 — APPROVED** [review]
Code looks fine

File: Lua Files/control.lua [hunk 1]
```diff
@@ -5,6 +5,10 @@ local tas_generator = require("steps")
 local steps = tas_generator.steps
 local debug_state = require("goal")
 local run = true
+local never_stop = false
+local use_all_ticks = false
+local step_executed = false
+local not_same_step = 1
```
Comment (theis999) [inline Lua Files/control.lua:11]: not_same_step does not appear to be in a global variable so it is not persistent across save/loads

File: Lua Files/control.lua [hunk 2]
```diff
@@ -1414,7 +1459,24 @@ script.on_event(defines.events.on_tick, function(event)
 	if compatibility_mode then
 		backwards_compatibility()
 	else
+		step_executed = false
 		handle_tick()
+
+		if global.use_all_ticks_warning_mode and step_executed then
```
Comment (theis999) [inline Lua Files/control.lua:1466]: step_executed is set to false 2 lines above, so this always evaluates to false

File: Lua Files/control.lua [hunk 3]
```diff
@@ -1414,7 +1459,24 @@ script.on_event(defines.events.on_tick, function(event)
...
+			end_use_all_ticks_warning_mode()
+		end
+
+		if use_all_ticks and not step_executed and global.use_all_ticks_warning_mode == nil and not player.mining_state.mining then
```
Comment (theis999) [inline Lua Files/control.lua:1470]: Same here, but inverse

File: Lua Files/control.lua (hunk 2 above)
Comment (MortenTobiasNielsen) [reply Lua Files/control.lua:1466]: Step executed is changed in handle_tick (in the different branches, which has something to do with executing a step).

File: Lua Files/control.lua (hunk 3 above)
Comment (MortenTobiasNielsen) [reply Lua Files/control.lua:1470]: Step executed is changed in handle_tick (in the different branches, which has something to do with executing a step).

File: Lua Files/control.lua (hunk 1 above)
Comment (MortenTobiasNielsen) [reply Lua Files/control.lua:11]: I think I have added it now - is it correctly done? :)

File: Lua Files/control.lua (hunk 2 above)
Comment (theis999) [reply Lua Files/control.lua:1466]: I see. That wasn't obvious 👍

File: Lua Files/control.lua (hunk 1 above)
Comment (theis999) [reply Lua Files/control.lua:11]: It looks correct 👍

**Pull request merged**